client/accounts.go

```diff
@@ -37,7 +37,40 @@ func (impl AccountsImpl) GetAccount(ctx context.Context, address string, opts ..
 
 type AccountResource struct {
 	Type string
-	Data interface{}
+	Data struct {
+		*CollectionsResource
+		*TokenStoreResource
+	}
+}
+
+type Table struct {
+	Handle string `json:"handle"`
+}
+
+type EventHandle struct {
+	Counter string
+	GUID    struct {
+		ID struct {
+			Addr        string `json:"addr"`
+			CreationNum string `json:"creation_num"`
+		} `json:"id"`
+	} `json:"guid"`
+}
+
+type CollectionsResource struct {
+	CollectionData    Table       `json:"collection_data"`
+	TokenData         Table       `json:"token_data"`
+	MintTokenEvents   EventHandle `json:"mint_token_events"`
+	CreateTokenEvents EventHandle `json:"create_token_events"`
```

Comment:
In my test, I got the field named `create_token_data_events` instead of `create_token_events` and `create_collection_events`